### PR TITLE
Quick Win Issue Template

### PR DESCRIPTION
**Pull request by Itay4**
## Status
- [x] Ready

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/quick-wins.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+about: Quick Win tasks to be taken by the community.
+title: ''
+labels: good first issue
+assignees: ''
+
+---
+
+Issues should only be opened by the repository maintainers. 
+
+Bugs should be reported to customer support and feature requests should be posted to AHA.
```
Comment (ShahafBenYakir) [inline .github/ISSUE_TEMPLATE/quick-wins.md:12]: Link to AHA?

File: .github/ISSUE_TEMPLATE/quick-wins.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+labels: good first issue
+assignees: ''
+
+---
+
+Issues should only be opened by the repository maintainers. 
+
+Bugs should be reported to customer support and feature requests should be posted to AHA. 
+
+Please do not open this issue.
```
Comment (ShahafBenYakir) [inline .github/ISSUE_TEMPLATE/quick-wins.md:14]: Not clear

**Review by ShahafBenYakir — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/quick-wins.md (hunk 1 above)
Comment (bakatzir) [inline .github/ISSUE_TEMPLATE/quick-wins.md:12]:
```suggestion
Bugs should be reported to customer support and feature requests should be created in [AHA](https://xsoar.ideas.aha.io/). 
```

File: .github/ISSUE_TEMPLATE/quick-wins.md (hunk 2 above)
Comment (Itay4) [reply .github/ISSUE_TEMPLATE/quick-wins.md:14]: not sure if you're serious, how can we be more clear than that?

**Review by bakatzir — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/quick-wins.md (hunk 2 above)
Comment (ShahafBenYakir) [reply .github/ISSUE_TEMPLATE/quick-wins.md:14]: I was actually serious lol. Why not put this at the beginning and say something like: Please do not open this issue - this is for internal usage only.

Comment (Itay4) [reply .github/ISSUE_TEMPLATE/quick-wins.md:14]: this is pretty much what it says, would you like to rephrase it ?

**Review by anara123 — APPROVED** [review]

**Comment by Itay4** [comment]
ci build fails on non related connectivity failures 

merging

**Pull request merged**